doc/changelog/08-tools/11230-timing-fuzz.rst

```diff
@@ -0,0 +1,15 @@
+- **Added:**
+  The ``make-both-single-timing-files.py`` script now accepts a
+  ``--fuzz=N`` parameter on the command line which determines how many
+  characters two lines may be offset in he "before" and "after" timing
+  logs while still being considered the same line.  When invoking this
+  script via the ``print-pretty-single-time-diff`` target in a
+  ``Makefile`` made by ``coq_makefile``, you can set this argument by
+  passing ``TIMING_FUZZ=N`` to ``make``.  (`#11230
+  <https://github.com/coq/coq/pull/11230>`_, by Jason Gross).
+
+- **Added:**
+  Coq's build system now supports both ``TIMING_FUZZ`` and
+  ``TIMING_SORT_BY`` just like a ``Makefile`` made by
+  ``coq_makefile``.  (`#11230
+  <https://github.com/coq/coq/pull/11230>`_, by Jason Gross).
```

Comment:
It looks to me like this is the only place where this second change is documented. Can you add some documentation for this in `INSTALL` for instance?

BTW, this looks like one more thing where we're going to have a regression (or need to port) when we switch to Dune.

Reply:
I've added a bit of documentation in `INSTALL`.

Indeed, we should port this to Dune.  I think we need a Dune expert (@ejgallego?) to implement support for `TIMED=1` (per-file timing on stdout by prefixing each command with `/usr/bin/env time -f $* (real: %e, user: %U, sys: %S, mem: %M ko)` on Linux and `gtime -f $* (real: %e, user: %U, sys: %S, mem: %M ko)` if `time -f` doesn't work) and `TIMING=1` (per-line times emitted for each file by passing `-time` to `coqc` and redirecting stdout to a `.timing` file for each `.v` file).  I can probably muddle through figuring out how to make the other targets based on those two, but I imagine that me and @ejgallego could figure it out much more quickly working in concert.